### PR TITLE
Okta 417576 self reg aspnet rework steps

### PR DESCRIPTION
**Pull request by robertnicolo-okta**
## Description:
Updated Step 18 and 19 based on description in the Jira Ticket.  Note that the heading titles had been changed in a previous change and are preserved.  However, the numbering and overall structure is per the Jira ticket. 

### Resolves:

* [OKTA-423082](https://oktainc.atlassian.net/browse/OKTA-423082)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 1]
```diff
@@ -424,8 +424,6 @@ switch (skipSelectionResponse.AuthenticationStatus)
 
 The method `SkipAuthenticatorSelectionAsync` can return these different response statuses: `Success` or `Terminal`. For a `Success` status, the user is signed in, the response is stored in session, and the user is redirected to the default sign-in page. In the csample app, the default sign-in page is the user profile page.
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:425]: Again, slightly awkward phrasing here. I'd replace "can return these different response statuses: `Success` or `Terminal`." with "can return a response status of `Success` or `Terminal`."

Also, you don't explain what the Terminal status means; only Success.

**Review by chrisdmills-okta — CHANGES_REQUESTED** [review]
Hey @robertnicolo-okta ! This is looking much better — I like these changes. I just had a few small asks in the interspersed comments, plus a couple of change requests on the first updated diagram:

There's a couple of typos in the "Diagram 3, Option 1" image:

* "Selecs the phone" -> should be "Select the phone"?
* "code field and submits" -> should be "code field and submit"?

Also, it seems to me like this diagram should have a "Continue flow in next diagram" labeled arrow at the bottom of it, to show that the overall flow is continued in the next diagram.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 2]
```diff
@@ -273,136 +273,136 @@ The remaining authenticator should display the phone factor to the user. Since t
...
-T)
+   After the user enters the phone code and clicks verify, a call is made to `VerifyAuthenticatorAsync`. In the phone verification use case, the code that passes into `VerifyAuthenticatorAsync` is the code that was sent through SMS to the phone number.
 
-```csharp
-var idxAuthClient = new IdxClient(null);
-           var verifyAuthenticatorOptions = new VerifyAuthenticatorOptions
-           {
-               Code = code,
-           };
+   T)
```
[the remark this answers is not in the document]
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:370]: Typo removed.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 3]
```diff
@@ -273,136 +273,136 @@ The remaining authenticator should display the phone factor to the user. Since t
...
+            };
 
-### 18h, Option 1: Complete authentication
+   var authnResponse = await idxAuthClient.VerifyAuthenticatorAsync(verifyAuthenticatorOptions, (IIdxContext)Session["idxContext"]);
+   ```
 
-The next step is to handle the response from `VerifyAuthenticatorAsync`. If the phone SMS code was valid, the method should return an `AuthenticationStatus` of `Success`. This status signifies that no more factors (required or optional) are waiting to be enrolled and verified.
+1. Complete authentication
 
-If the steps described in[Set up your Okta org (for multifactor use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-multi-factor-use-cases) were properly followed, the user should now be registered with no more factors to be verified. The user should then be sent to the default page after they have successfully registered. In the sample application, the user is sent to the user profile page.
```
[the remark this answers is not in the document]
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:386]: space added.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 4]
```diff
@@ -273,136 +273,136 @@ The remaining authenticator should display the phone factor to the user. Since t
...
+            authnResponse.TokenInfo);
+            _authenticationManager.SignIn(new AuthenticationProperties(), identity);
+            return RedirectToAction("Index", "Home");
+      ...
+   }
+   ```
+
+#### Option 2: Skip phone enrollment
 
 If the user opts to skip phone enrollment, a call to `SkipAuthenticatorSelectionAsync` needs to be made. This method skips phone enrollment and eliminates the need to verify the factor. See the following code snippet for more details.
```
[the remark this answers is not in the document]
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:407]: Updated with "factor:"

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md (hunk 1 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:425]: I went back to the code to learn more about the Terminal status and it has been removed. In its place, there are now exceptions. I've updated the content to reflect the new code changes.

**Review by chrisdmills-okta — APPROVED** [review]
Nice work @robertnicolo-okta ! This is all looking good now.

**Pull request merged**